deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -134,8 +144,8 @@ service:
       receivers: [otlp]
       processors: [batch]
       exporters: [clickhousemetricswrite]
-    metrics/hostmetrics:
-      receivers: [hostmetrics]
+    metrics/generic:
+      receivers: [hostmetrics, prometheus]
       processors: [resourcedetection, batch]
```

Comment:
I will pull these changes and take a look at this locally. Prometheus adds `job_name` as `service.name`, and you have `resourcedetection`, creating a conflicting key value. I also realised `override: false` is only fixed in charts, not here.

Reply:
This is fine. It uses the job name because the job -> service_name mapping is done after the resource is built, and we are using the last one. But whatever inspired this change could have been solved by adding `resource_to_telemetry_conversion` in `collector-metrics-config`.